packages/nodes-base/nodes/Onfleet/descriptions/WorkerDescription.ts

```diff
@@ -0,0 +1,529 @@
+} as INodeProperties;
+
+const vehicleDescriptionField = {
+	displayName: 'Description',
+	name: 'description',
+	type: 'string',
+	default: '',
+	description: 'The vehicle\'s make, model, year, or any other relevant identifying details.',
+} as INodeProperties;
+
+const vehicleLicensePlateField = {
+	displayName: 'License plate',
+	name: 'licensePlate',
+	type: 'string',
+	default: '',
+	description: 'The vehicle\'s license plate number.',
+} as INodeProperties;
+
+const vehicleColorField = {
+	displayName: 'Color',
+	name: 'color',
+	type: 'string',
+	default: '',
+	description: 'The vehicle\'s color.',
+} as INodeProperties;
+
+const teamsField = {
+	displayName: 'Teams (JSON)',
+	name: 'teams',
+	type: 'json',
+	default: '[]',
+	description: 'One or more team IDs of which the worker is a member.',
+} as INodeProperties;
+
+const teamsFilterField = {
+	displayName: 'Teams',
+	name: 'teams',
+	type: 'string',
+	default: '',
+	description: 'A comma-separated list of the team IDs that workers must be part of.',
+} as INodeProperties;
+
+const statesFilterField = {
+	displayName: 'States',
+	name: 'states',
+	type: 'multiOptions',
+	options: [
+		{
+			name: 'Off-duty',
+			value: 0,
+		},
+		{
+			name: 'Idle (on-duty, no active task)',
+			value: 1,
+		},
+		{
+			name: 'Active (on-duty, active task)',
+			value: 2,
+		},
+	],
+	default: '',
+	description: 'List of worker states.',
+} as INodeProperties;
+
+const phonesFilterField = {
+	displayName: 'Phones',
+	name: 'phones',
+	type: 'string',
+	default: '',
+	description: 'A comma-separated list of workers\' phone numbers.',
+} as INodeProperties;
+
+const filterField = {
+	displayName: 'List of fields to return',
+	name: 'filter',
+	type: 'string',
+	default: '',
+	description: 'A comma-separated list of fields to return, if all are not desired. For example, name, location.',
+} as INodeProperties;
+
+const longitudeFilterField = {
+	displayName: 'Longitude',
+	name: 'longitude',
+	type: 'number',
+	typeOptions: {
+		numberPrecision: 14,
+	},
+	default: '',
+	description: 'The longitude component of the coordinate pair.',
+} as INodeProperties;
+
+const latitudeFilterField = {
+	displayName: 'Latitude',
+	name: 'latitude',
+	type: 'number',
+	typeOptions: {
+		numberPrecision: 14,
+	},
+	default: '',
+	description: 'The latitude component of the coordinate pair.',
+} as INodeProperties;
+
+const radiusFilterField = {
+	displayName: 'Radius',
+	name: 'radius',
+	type: 'number',
+	typeOptions: {
+		maxValue: 10000,
+		minValue: 0,
+	},
+	default: 1000,
+	description: 'The length in meters of the radius of the spherical area in which to look for workers. Defaults to 1000 if missing. Maximum value is 10000.',
+} as INodeProperties;
+
+const scheduleDateField = {
+	displayName: 'Date',
+	name: 'date',
+	type: 'dateTime',
+	default: Date.now(),
+	description: 'Schedule\'s date.',
+} as INodeProperties;
+
+const scheduleTimezoneField = {
+	displayName: 'Timezone',
+	name: 'timezone',
```

Comment:
Load all timezones from moment.

			async getTimezones(
				this: ILoadOptionsFunctions,
			): Promise<INodePropertyOptions[]> {
				const returnData: INodePropertyOptions[] = [];
				for (const timezone of moment.tz.names()) {
					const timezoneName = timezone;
					const timezoneId = timezone;
					returnData.push({
						name: timezoneName,
						value: timezoneId,
					});
				}
				return returnData;
			},